src/luaotfload-database.lua

```diff
@@ -2239,7 +2237,7 @@ local truncate_string = function (str, restrict)
     local len = utf8len (str)
     if wd - len < 0 then
         --- combined length exceeds terminal,
-        str = ".." .. stringsub(str, len - wd + 2)
```

Comment:
`len` is calculated in Unicode codepoints, but stringsub needs it's argument in bytes. So this is only correct if `str` is pure ASCII, but then `utf8len` is kind of overkill. With `utf8offset` the problem disappears.